### PR TITLE
Crashed UPP Survivors Insert Tweaks

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Changes based on feedback and personal observations

Major changes
- A second map insert has been added that is guaranteed to occur whenever UPP survivors spawn. A second, destroyed, UPP dropship containing supplies (10 standard t-71 mags, 4 AP mags, a bunch of grenades). Including a single UPP PKP Machinegun. Only the Specialist and SL can use it. (I'd say its balanced because the weapon has horrible accuracy and will likely cause FF in unskilled hands). 
- UPP SL spawns with a guaranteed set of armour and their beret instead of a officer's jacket. Feels more thematic and the officer's jacket is meant for high ranking officers and not SLs. 
- Reduced spawns so there is only 1 Medic and 1 Engineer spawn, but they are both now a higher priority to spawn. Spawns should now go SL > Spec > Engi / Medic > Soldiers.
- SL and Spec now have driving skills.  
- Added a lore entry when the survs spawn to give survs a hint to go explore to find the other crashed ship.

Minor Changes
- Decreased chance of a UPP spawning with a Pulse Carbine slightly, due to how weak it is. Now 1/5 chance instead of 1/3.
- Adds some buckshot ammo to the crashed dropship due to low ammo amount given to UPP who spawn with shotgun. UPP Shotgun feels balanced since it has high upfront damage but a low RoF. However low ammo meant a lot of people were ditching them for survivor shotguns. 
- SL has UPP spec weapon training (in case a synth takes the Spec spawn, so at least 1 person can use the minigun)
- UPP now have access to the UPP comms channel. 
- HUD icons are now in place. 

# Explain why it's good for the game

The Medic/Engi changes ensure there is a guarantee of a single Medic and Engineer outside of very lowpop. I was noticing the prior spawn system means you'll see a lot of Medics but rarely a Engineer. This change should also mean more people will spawn as Soldiers instead of 90% of the squad being Medics and Engineers. 

The supply cache is to give the UPP a alternative objective, instead of rushing to the meta hold spots they can try to make a run for a supply cache to give them a better chance of survival.

Considering my observation of as many UPP surv rounds as I can, I've found its almost guaranteed the UPP are overrun by the time the USCM arrive. Perhaps a chance at some extra ammo might tip the scales? 

Everything else is self explanatory. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Screenshot 2023-08-15 00 20 09

Screenshot 2023-08-15 01 41 24

</details>

# Changelog
:cl:
balance: Large amount of balance changes to UPP survivors on Trijent. New spawn setup guarantee's you'll get 1 Medic and 1 Engineer. 
balance: UPP survivors now have access to HUD icons for squad roles, and access to the UPP communications channel if the colony communications is repaired. 
balance: UPP surv SL's and Specialists can now drive vehicles. 
balance: A second, crashed, UPP dropship can be found somewhere on Trijent if the UPP survivors spawn. It contains extra supplies and stronger firearms. But requires a detour to find. 
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
Ok, I think we're good.

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -1022,3 +1022,13 @@
 	initial_keys = list(/obj/item/device/encryptionkey/vc)
 	volume = RADIO_VOLUME_RAISED
 	multibroadcast_cooldown = HIGH_MULTIBROADCAST_COOLDOWN
+
+/obj/item/device/radio/headset/distress/UPP/recon
+	name = "UPP headset"
```
Comment (Zonespace27) [inline code/game/objects/items/devices/radio/headset.dm:1027]:
```suggestion
	name = "\improper UPP headset"
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
get a mapper to sign off and i'll merge

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by morrowwolf — APPROVED** [review]
Looks good let's gooo!

**Pull request merged**